x-pack/plugins/maps/public/kibana_services.ts

```diff
@@ -11,16 +11,22 @@ import type { MapsPluginStartDependencies } from './plugin';
 import type { EMSSettings } from '../../../../src/plugins/maps_ems/common/ems_settings';
 import type { PaletteRegistry } from '../../../../src/plugins/charts/public';
 import { MapsEmsPluginPublicStart } from '../../../../src/plugins/maps_ems/public';
+import { plugin, SharedUXPluginStart } from '../../../../src/plugins/shared_ux/public';
 
 let coreStart: CoreStart;
 let pluginsStart: MapsPluginStartDependencies;
 let mapsEms: MapsEmsPluginPublicStart;
 let emsSettings: EMSSettings;
+let sharedUXPluginContext: SharedUXPluginStart;
+
 export function setStartServices(core: CoreStart, plugins: MapsPluginStartDependencies) {
   coreStart = core;
   pluginsStart = plugins;
   mapsEms = plugins.mapsEms;
   emsSettings = mapsEms.createEMSSettings();
+
+  const sharedUXPlugin = plugin();
```

Comment:
This is not the correct way to consume a plugin. remove lines 20, 28, 29. 

Instead, 
1) In maps/kibana.json, add "shared_ux" to "requiredPlugins" array.
2) In maps/public/plugin.ts, add "shared_ux" to "MapsPluginStartDependencies" interface.
3) In maps/plugin/kibana_services, getSharedUXPluginContext should just return `pluginsStart.sharedUx.ServicesContext`.